openlibrary/core/vendors.py

```diff
@@ -292,8 +292,8 @@ def _get_betterworldbooks_metadata(isbn):
             qlt = 'used'
 
         if new_qty and new_qty[0] and new_qty[0] != '0':
-            _price = used_price[0] if used_price else None
-            if price and _price and _price < price:
+            _price = new_price[0] if new_price else None
+            if _price and (not price or _price < price):
```

Comment:
`_price` and `price` are strings here

Reply:
It'll work in some cases, but here it'll show the $30.00 new price, instead of the $9.00 used:
```
'30.00' and (not '9.00' or '30.00' < '9.00')
True
```

Reply:
fixed